tests/fixtures/migrations/foreign-key-on-delete.php

```diff
@@ -17,7 +17,7 @@ public function up()
 
         Schema::create('comments', function (Blueprint $table) {
             $table->id();
-            $table->foreignId('site_id')->constrained()->cascadeOnDelete();
+            $table->foreignId('site_id')->constrained();
```

Comment:
Why is this one being removed?

Reply:
Because that's how it's defined in the draft file. P.S. `config('blueprint.use_constraints') === false `

https://github.com/laravel-shift/blueprint/blob/e966567125b5db484af1acdbd212ebeeb67adcdd/tests/fixtures/drafts/foreign-key-on-delete.yaml#L1-L5

Reply:
Gotcha. I guess I would still want to see a case for `delete` + `cascade` option. I worry we lost some coverage.

Reply:
no worries, pushing changes now.